### ISSUE TITLE
Add Container Annotations profile for all collections

### ISSUE THREAD

**Issue by nicholascar** [RDFLib/prez#18]
e.g. SpacePrez' list of Datasets: https://linked.fsdf.org.au/datasets, the Collections in a Dataset, https://linked.fsdf.org.au/dataset/electrical-infrastructure/collections, the Features within a Collection, https://linked.fsdf.org.au/dataset/electrical-infrastructure/collections/PS/items.

e.g. VocPrez' list of Vocabs, https://linked.fsdf.org.au/vocab, a Vocab (Concepts) https://linked.fsdf.org.au/vocab/observable-properties and any SKOS Collections views, https://vocabs.gsq.digital/object?uri=http%3A//linked.data.gov.au/def/data-access-rights/open-access-rights&_profile=alt

Basically, wherever a _Members_ profile is currently present

**Comment by recalcitrantsupplant** [comment]
what are the properties that would be included in the annotations profile?
